app/views/projects/samples/attachments/_attachment.html.erb

```diff
@@ -15,8 +15,52 @@
     id: dom_id(attachment.file),
     "aria-label": attachment.file.filename,
     data: {
-      action: "input->selection#toggle",
-      selection_target: "rowSelection"
+      "projects--samples--attachments--table-target": "input",
+      action: "input->projects--samples--attachments--table#toggle",
+      name:
+        (
+          if attachment.associated_attachment
+            "<div class='flex items-center'>#{
+              viral_icon(
+                name: :arrow_right,
+                color: :subdued,
+                classes: "h-6 w-6 ml-0 mr-2"
+              )
+            }<span>#{attachment.file.filename}</span></div>".html_safe
+          else
+            attachment.file.filename
+          end
+        ),
+      attachment_name:
```

Comment:
Just for clarity, can  we change the `name` and `size` to `attachment_name` and `attachment_size` for the first attachment and `attachment_name`, and `attachment_size` for the associated attachment to `associated_attachment_name` and `associated_attachment_size`. This will require an update in I believe just the `concatenations_files_controller.js` as well

Reply:
1da8cd8215fb1a806949584eb1b27c84cbe8dfe8